primefaces/src/main/java/org/primefaces/component/datatable/DataTableBase.java

```diff
@@ -67,6 +67,7 @@ public enum PropertyKeys {
         filterEvent,
         filteredValue,
         frozenColumns,
+        alignFrozenColumnsRight,
```

Comment:
rename this property to `frozenColumnsAlignment` and the values will be "right" or "left" with the default "left"

Then in the widget you can make a boolean like `var isFrozenColumnsRight = this.cfg.frozenColumnsAlignment === "right";`